runtime/vm/lookupmethod.c

```diff
@@ -1135,15 +1120,18 @@ illegalAccessMessage(J9VMThread *currentThread, IDATA badMemberModifier, J9Class
 		}
 	} else if (J9_VISIBILITY_NON_MODULE_ACCESS_ERROR != errorType) {
 		/* illegal module access */
+		J9JavaVM const *const vm = currentThread->javaVM;
```

Comment:
Instead of cache vm, you can just cache currentThread->javaVM->modulePointerOffset to a local variable.